exampleSite/config.yml

```diff
@@ -129,6 +129,12 @@ params:
       link: https://example.com/default
       #description: not there either
 
+  certificates:
```

Comment:
Well this case ... this should be certifications too :D 

think this is the last one, already tested and looks good. :D

Reply:
There's always a variable or a semi colon somewhere that would throw a error! updated the config.yml